lib/assets/SnapshotAlertManager.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  SnapshotAlertManager.swift
+//
+//  Created by Maxime Britto on 06/01/2016.
+//  Copyright © 2016 dev2a. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+func automaticallyDismissAlerts(testCase:XCTestCase) -> NSObjectProtocol {
+    return SnapshotAlertManager.automaticallyDismissAlerts(testCase)
+}
+
+func stopDismissingAlerts(testCase:XCTestCase) {
+    SnapshotAlertManager.stopDismissingAlerts(testCase)
+}
+
+func waitForAlertsToBeDismissed() {
+    SnapshotAlertManager.waitForAlertsToBeDismissed()
+}
+
+class SnapshotAlertManager: NSObject {
+    static var alertInterruptionMonitor:NSObjectProtocol?
+
+    class func stopDismissingAlerts(testCase:XCTestCase) {
+        if let previousDismissHandler = alertInterruptionMonitor {
+            testCase.removeUIInterruptionMonitor(previousDismissHandler)
+        }
+    }
+
+    class func automaticallyDismissAlerts(testCase:XCTestCase) -> NSObjectProtocol
+    {
+        SnapshotAlertManager.stopDismissingAlerts(testCase)
+        alertInterruptionMonitor = testCase.addUIInterruptionMonitorWithDescription("Snapshot alert auto hide") { (alert: XCUIElement) -> Bool in
```

Comment:
It is one of those undocumented functions... :( 
I've seen many code examples which were manually touching one button by typing its title and then returning true to indicate the event had been consumed. But if you do nothing and return false the alert one button is clicked automatically. I've tested it on one of my own alerts with only one button and it was clicked and for system alerts (push notification user authorization) it is the 'OK' button that is clicked for me.